flink-rpc/flink-rpc-akka/src/main/java/org/apache/flink/runtime/rpc/pekko/PekkoRpcServiceUtils.java

```diff
@@ -78,13 +78,9 @@ static PekkoRpcService createRemoteRpcService(
             throws Exception {
         final PekkoRpcServiceBuilder rpcServiceBuilder =
                 PekkoRpcServiceUtils.remoteServiceBuilder(
-                        configuration, externalAddress, externalPortRange);
-
-        if (bindAddress != null) {
-            rpcServiceBuilder.withBindAddress(bindAddress);
-        }
-
-        bindPort.ifPresent(rpcServiceBuilder::withBindPort);
+                                configuration, externalAddress, externalPortRange)
```

Comment:
this changes behaviour - prior to the fix if there was no bindPort we would build with the bind address if there was one.

Reply:
In fact, create `ActorSystem` without `bindPort` is legal.